DeviceAdapters/Zaber/ObjectiveChanger.cpp

```diff
@@ -0,0 +1,366 @@
+///////////////////////////////////////////////////////////////////////////////
+// FILE:          ObjectiveChanger.cpp
+// PROJECT:       Micro-Manager
+// SUBSYSTEM:     DeviceAdapters
+//-----------------------------------------------------------------------------
+// DESCRIPTION:   Device adapter for Zaber's X-MOR objective changer.
+//
+// AUTHOR:        Soleil Lapierre (contact@zaber.com)
```

Comment:
```suggestion
// AUTHOR:        Martin Zak (contact@zaber.com)
```

We must make sure the blame goes to the right place. :)